charts/lighthouse/templates/tekton-controller-role.yaml

```diff
@@ -5,50 +5,33 @@ metadata:
   name: {{ template "tektoncontroller.name" . }}
 rules:
 - apiGroups:
-  - ""
+  - tekton.dev
   resources:
-  - namespaces
-  - configmaps
-  - secrets
+  - pipelines
   verbs:
   - get
-  - list
-  - watch
 - apiGroups:
   - tekton.dev
   resources:
-  - pipelineresources
-  - tasks
-  - pipelines
   - pipelineruns
   verbs:
   - create
   - list
   - get
-  - update
   - watch
-  - patch
-  - delete
 - apiGroups:
   - lighthouse.jenkins.io
   resources:
   - lighthousejobs
   verbs:
-  - create
-  - delete
-  - list
-  - update
```

Comment:
Looks like we still need this -

```
{"component":"lighthouse-tekton-controller","controller":"tekton-controller","file":"/workspace/source/pkg/engines/tekton/controller.go:131","func":"github.com/jenkins-x/lighthouse/pkg/engines/tekton.(*LighthouseJobReconciler).Reconcile","level":"error","msg":"failed to update Project status: lighthousejobs.lighthouse.jenkins.io \"700713fa-e887-11ea-bba5-f6ebb3fd55d9\" is forbidden: User \"system:serviceaccount:lh-test:lighthouse-tekton-controller\" cannot update resource \"lighthousejobs\" in API group \"lighthouse.jenkins.io\" in the namespace \"lh-test\"","time":"2020-08-27T17:05:35Z"}
```

Specifically, it's https://github.com/jenkins-x/lighthouse/blob/8fe7a143c8b56ad75c6a7acbd6dcdf2944d17b58/pkg/engines/tekton/controller.go#L127-L131 - we're probably being too aggressive in that update call (no need to make the update if we haven't actually changed the label, e.g.), but we do need update rights.